reference/commands/inspect.rst

```diff
@@ -23,3 +23,127 @@ conan inspect
                             -vvv or -vtrace
       --logger              Show the output with log format, with time, type and
                             message.
+
+Inspect default and json output
+-------------------------------
+
+This ``conan inspect`` default output shows the following information format.
```

Comment:
It is worth adding an "important" note that says that ``conan inspect`` doesn't really evaluate any methods or apply any conditional logic. It list class attributes only.